packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -116,24 +117,34 @@ export interface IFluidContainer extends IEventProvider<IFluidContainerEvents> {
     /**
      * A newly created container starts detached from the collaborative service.  Calling attach() uploads the
      * new container to the service and connects to the collaborative service.
+     *
+     * TODO: what states can this be called in? Just in "Disconnected"?
+     *
      * @returns A promise which resolves when the attach is complete, with the string identifier of the container.
      */
     attach(): Promise<string>;
 
     /**
-     * Attempts to connect the container to the delta stream and process ops
+     * Attempts to connect the container to the delta stream and process ops.
+     *
+     * TODO: what states can this be called in? Just in "ConnectionState.Disconnected"?
```

Comment:
```suggestion
     * This should only be called when the container is in the "ConnectionState.Disconnected" state. This can be determined by observing the `connectionState`.
```